src/ontology/uberon-edit.obo

```diff
@@ -224246,6 +224246,119 @@ intersection_of: part_of UBERON:0001987 ! placenta
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-06-27T09:30:52Z xsd:dateTime
 
+[Term]
+id: UBERON:8480009
+name: perianal space
+def: "The virtual spaces circumscribed by the mucosa lining of the anal canal and the muscular structures of the sphincter complex." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:0000464 ! anatomical space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0001353 ! anal region
+property_value: dcterms-date 2022-07-05T10:24:30Z xsd:dateTime
+
+[Term]
+id: UBERON:8480010
+name: subcutaneous perianal space
+def: "A space located below the dentate line. Its borders are the skin of the anal verge below and the internal sphincter above." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:36:29Z xsd:dateTime
+
+[Term]
+id: UBERON:8480011
+name: submucous perianal space
+def: "A space located above the dentate line. It is delimited by the internal sphincter and by the anoderm." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
+is_a: UBERON:8480009 ! perianal space
+relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
+property_value: dcterms-date 2022-07-05T10:41:02Z xsd:dateTime
+
+[Term]
+id: UBERON:8480012
+name: intersphincteric perianal space
+def: "A space located between the internal and external anal sphincter." [http://www.endoscopy-colon-explorer.com/anatomy-of-the-anal-region/]
```

Comment:
"A perianal space located..."

Reply:
I'm not sure about the website, but I generally prefer the xref to be something I know is permanent. This prevents problems if the website closes for example. Totally understand if you want to keep it as that though, just something for you to think about.

Reply:
yes, I understand and agree. I will try to see if I find something in literature. The website contains pictures that help also to understand all these holes

Reply:
"A perianal space located between the internal and external anal **sphincter**."

I think this should be "sphincters".

Reply:
@bvarner-ebi I corrected to  "sphincters". Thanks

Reply:
@bvarner-ebi @shawntanzk I renamed this term "intersphincteric space" to align names defined in https://www.ncbi.nlm.nih.gov/books/NBK535387/